File: use-cases/merchant-fulfillment/code/java/src/main/java/lambda/utils/OrderChangeNotification.java
package lambda.utils;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class OrderChangeNotification {

    @JsonProperty("NotificationLevel")
    public String notificationLevel;

    @JsonProperty("AmazonOrderId")
    public String amazonOrderId;

    @JsonProperty("Summary")
    public NotificationOrderSummary summary;
}

File: use-cases/solicitations/code/csharp/src/sp-api-csharp-app/lambda/utils/OrderChangeNotification.cs
namespace SpApiCsharpApp.lambda.utils
{
    public class OrderChangeNotification
    {
        public String NotificationLevel { get; set; }

        public String SellerId { get; set; }

        public String AmazonOrderId { get; set; }

        public NotificationOrderSummary Summary { get; set; }
    }
}